lib/core/test/unit/Cardano/Wallet/DB/Properties.hs

```diff
@@ -308,15 +320,37 @@ properties = do
                 , 2710 , 2711 , 2712 , 2713 , 2714
                 ]
 
+        it "k=2160, h=2414" $ \_ -> do
+            let cfg = SparseCheckpointsConfig
+                    { gapsSize = 100
+                    , edgeSize = 0
+                    }
+            let k = Quantity 2160
+            let h = Quantity 2714
+            -- First unstable block: 554
+            sparseCheckpoints cfg k h `shouldBe`
+                [ 0    , 500  , 600  , 700  , 800  , 900
+                , 1000 , 1100 , 1200 , 1300 , 1400 , 1500
+                , 1600 , 1700 , 1800 , 1900 , 2000 , 2100
+                , 2200 , 2300 , 2400 , 2500 , 2600 , 2700
+                , 2714
+                ]
+
         it "The tip is always a checkpoint" $ \_ ->
             property prop_sparseCheckpointTipAlwaysThere
 
-        it "There's at least (min h 10) checkpoints" $ \_ ->
+        it "There's at least (min h edgeSize) checkpoints" $ \_ ->
             property prop_sparseCheckpointMinimum
 
         it "There's no checkpoint older than k (+/- 100)" $ \_ ->
             property prop_sparseCheckpointNoOlderThanK
 
+        it "All else equal, sparse checkpoints are the same for all edge size" $ \_ ->
```

Comment:
I think the description is more confusing than the actual property.

Could something like:

`∀ cfg. sparseCheckpoints (cfg { edgeSize = 0 }) ⊆ sparseCheckpoints cfg`

work?

(Might be slightly misleading with `L.isSubsequenceOf ` vs `⊆` though)

Reply:
Maths-like are more readable here, agreed.